src/middlewared/middlewared/alembic/versions/25.10/2025-09-24_21-11_tnc_prod_endpoints.py

```diff
@@ -0,0 +1,59 @@
+"""
+Moving tn_connect url from staging to production
+
+Revision ID: 53193fbee3ee
+Revises: 5ea9f662ced4
+Create Date: 2025-09-24 16:44:06.551471+00:00
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '53193fbee3ee'
+down_revision = '5ea9f662ced4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+
+    production_urls = {
+        'account_service_base_url': 'https://account-service.tys1.truenasconnect.net/',
+        'leca_service_base_url': 'https://dns-service.tys1.truenasconnect.net/',
+        'tnc_base_url': 'https://web.truenasconnect.net/',
+        'heartbeat_url': 'https://heartbeat-service.tys1.truenasconnect.net/'
+    }
+
+    result = conn.execute(
+        'SELECT id, account_service_base_url, leca_service_base_url, tnc_base_url, heartbeat_url FROM truenas_connect'
+    ).fetchall()
+
+    if not result:
+        return
+
+    row = result[0]
+    row_id = row[0]
+    current_urls = {
+        'account_service_base_url': row[1],
+        'leca_service_base_url': row[2],
+        'tnc_base_url': row[3],
+        'heartbeat_url': row[4]
+    }
+
+    all_non_prod = all(
```

Comment:
Is there ever a situation where some of the URLs are not staging? Another way to ask the question, why do we have the `all_non_prod` check? Why not just check the current entries and update them if they have `.dev` or `.staging` in the name?